cmd/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "linkschkr"
+
+func main() {
+	linkschkr.Run(1000, []string{"https://golang.org"})
```

Comment:
Seems like functional options would be useful here, doesn't it; at first sight `1000` means nothing to us. But something like:

```go
linkschkr.Run(
	linkschkr.WithWorkers(1000)
)
```

is much clearer.